### PR TITLE
[TE] Add data quality/sla rule to the detection template

### PR DESCRIPTION
**Pull request by akshayrai**
* Add sla rule to the detection onboarding template

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/utils/yaml-tools.js [hunk 1]
```diff
@@ -11,24 +11,29 @@ dataset: dataset_to_which_this_metric_belongs
...
+        mlConfig: true          # Automatically maintain configuration with the best performance.
+  filter:                       # Filter out anomalies detected by rules to reduce noise.
+  - name: filter_rule_1
+    type: PERCENTAGE_CHANGE_FILTER
+    params:
+      pattern: UP_OR_DOWN       # Other patterns: "UP","DOWN".
+      threshold: 0.05           # Filter out all changes less than 5% compared to baseline.
+  quality:                      # Configure the data quality rules
+  - name: data_sla_rule_1
+    type: DATA_SLA              # Alert if data is delayed
```
Comment (xiaohui-sun) [inline thirdeye/thirdeye-frontend/app/utils/yaml-tools.js:29]: Alert if data is missing.

File: thirdeye/thirdeye-frontend/app/utils/yaml-tools.js [hunk 2]
```diff
@@ -11,24 +11,29 @@ dataset: dataset_to_which_this_metric_belongs
...
+    params:
+      sla: 3_DAYS               # Data delayed by 3 days since last availability
```
Comment (xiaohui-sun) [inline thirdeye/thirdeye-frontend/app/utils/yaml-tools.js:31]: # Data is missing for 3 days.

File: thirdeye/thirdeye-frontend/app/utils/yaml-tools.js (hunk 1 above)
Comment (akshayrai) [reply thirdeye/thirdeye-frontend/app/utils/yaml-tools.js]: updated

File: thirdeye/thirdeye-frontend/app/utils/yaml-tools.js (hunk 2 above)
Comment (akshayrai) [reply thirdeye/thirdeye-frontend/app/utils/yaml-tools.js]: updated

**Pull request merged**